LayoutTests/crypto/subtle/X25519-generate-key-exctractable.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+<script src="../resources/common.js"></script>
+</head>
+<body>
+<p id="description"></p>
+<div id="console"></div>
+
+<script>
+description("Test generating an extractable EC key pair with Curve25519 using X25519 algorithm.");
+
+jsTestIsAsync = true;
+
+var extractable = true;
+
+debug("Generating a key pair...");
+crypto.subtle.generateKey({ name: "X25519", namedCurve: "Curve25519"}, extractable, ["deriveKey", "deriveBits"]).then(function(result) {
+    keyPair = result;
+    shouldBe("keyPair.toString()", "'[object Object]'");
+    shouldBe("keyPair.publicKey.type", "'public'");
+    shouldBe("keyPair.publicKey.extractable", "true");
+    shouldBe("keyPair.publicKey.algorithm.name", "'X25519'");
+    shouldBe("keyPair.publicKey.algorithm.namedCurve", "'Curve25519'");
```

Comment:
Neither key generation parameters nor the key algorithm should need/have a `namedCurve` property.